apps/web/src/contexts/globalLayout/GlobalAnnouncementPopover/components/FeaturedCollectorCard.tsx

```diff
@@ -11,7 +11,7 @@ import FollowButton from '~/components/Follow/FollowButton';
 import { FeaturedCollectorCardCollectionFragment$key } from '~/generated/FeaturedCollectorCardCollectionFragment.graphql';
```

Comment:
no screenshot for this one as it requires reinstating the GlobalAnnouncementPopover

will be easy enough to check if i messed anything up next time we work on that component